app/concerns/agent_controller_concern.rb

```diff
@@ -68,7 +68,8 @@ def control!
             log "Agent '#{target.name}' is disabled"
           end
         when 'configure'
-          target.update! options: target.options.deep_merge(interpolated['configure_options'])
+          new_options = target.options.deep_merge(interpolated['configure_options']) { |key, old, new| !(new.kind_of?(Array) && old.kind_of?(Array)) ? new : (old+new).uniq }
```

Comment:
If old and new are not arrays, how can we `(old+new).uniq`?  Should it be `Array(old) + Array(new)).uniq`?